apps/site/assets/ts/stop/components/DeparturesAndMap.tsx

```diff
@@ -60,15 +60,74 @@ const DeparturesAndMap = ({
     return false;
   };
 
+  const defaultPolylines = chain(routesWithPolylines)
+    .filter(
+      route =>
+        isASilverLineRoute(route) ||
+        isSubwayRoute(route) ||
+        isACommuterRailRoute(route)
```

Comment:
question: Why are we filtering out the Silver Line, Subway, and Commuter Rail lines?

Reply:
Okay, I got confused how `filter` works.  I thought it removed everything the predicate was true for.